test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -734,5 +835,10 @@ public static IEnumerable<object[]> GetPackagesConfigTemplates()
         {
             yield return new object[] { ProjectTemplate.ClassLibrary };
         }
+
+        public static IEnumerable<object[]> GetIOSTemplates()
+        {
+            yield return new object[] { ProjectTemplate.IOSLibraryApp };
```

Comment:
What kind of project is this creating?
Is it a xamarin.ios project?

Reply:
Yes, we want to create a xamarin.ios project, We have added "compent.xamarin" , "ios" and "runtimes.ios" workloads into test.vsconfig, but the project still can't be created successfully, could you tell us which workloads need to be installed for this project? Thanks.

Reply:
Can you maybe try to reverse engineer that?

Start by installing a Visual Studio that *can* create the ios projects and then export the vsconfig https://learn.microsoft.com/en-us/visualstudio/install/import-export-installation-configurations?view=vs-2022 and then update the in repo one to match that.

Reply:
Hi @nkolev92, yes that's how we investigate that issue. We are adding the probable workloads one by one after getting them from "export the vsconfig" to see which one will work.

Reply:
@v-luzh I think Nikolche's suggest was to do something like:

1. Create a VM and install VS using only the components listed in `tests.dev17.vsconfig`
2. Try to run the tests on the machine
3. If the tests failed, then add components in the VS installer, and repeat step 2
4. Once the tests pass locally, then you can export the update `tests.dev17.vsconfig` in the repo.

This will allow you to iterate much more quickly than running CI jobs over and over.

Reply:
Hi @zivkan，thanks for your reminder. When we met the workload issue last week, we still had the local-debugging problem then. Nikolche just fixed the problem three days ago and we are still study how to do it on local machine (following his instruction from PR [5124](https://github.com/NuGet/NuGet.Client/pull/5124)) now.

Now the project can be created successfully and the test can pass as the running result of the build 6.7.0.50 after changing the assert method to "AssertPackageInAssetsFile"/"AssertPackageNotInAssetsFile". Please help to approve it, thanks.